frontends/web/src/app.tsx

```diff
@@ -60,7 +60,7 @@ class App extends Component<Props, State> {
     if (panelStore.state.activeSidebar) {
       toggleSidebar();
     }
-  }
+  };
```

Comment:
At first I was confused why it added a semicolon here, but found this https://github.com/prettier/prettier/issues/1444#issuecomment-297835274

> class properties - unlike method declarations in classes - do have semicolons after them.